index.html

```diff
@@ -489,7 +489,7 @@
           var last = points[points.length - 2]; // - 2 because - 1 is the hovering point
 
           // End the wire if you connected with the first, or clicked the same point twice:
-          if (first.x === x && first.y === y || last.x === x && last.y === y) {
+          if ((first.x === x && first.y === y) || (last.x === x && last.y === y)) {
```

Comment:
This is actually adding unnecessary formatting / syntax. But I agree with your plugin, this is more clear ;)